### DISCUSSION TITLE
I'm unable to install python-torch

### DISCUSSION THREAD

**Discussion by GUUser91** [termux/termux-packages#15193] [Q&A] [answered]
I can't install python-torch. I even added a Grimler mirror. And I update with pkg update. 
https://github.com/termux/termux-packages/wiki/Mirrors
Even with a updated repo. I get this error mesage.
> E: Unable to locate package python-torch

**Comment by sylirre** [comment, marked as answer]
This package has been removed because support of Python 3.11 was unavailable as of Oct 30, 2022.

When the latest stable version will have Python 3.11 support, `python-torch` will be enabled again.
